sdk/formrecognizer/tests.yml

```
@@ -1,104 +1,46 @@
-                  CoverageArg: '--disablecov'
-                Windows_Python27:
-                  Pool: azsdk-pool-mms-win-2019-general
-                  OSVmImage: MMS2019
-                  PythonVersion: '2.7'
-                  CoverageArg: '--disablecov'
-                Linux_Python39:
-                  Pool: azsdk-pool-mms-ubuntu-1804-general
-                  OSVmImage: MMSUbuntu18.04
-                  PythonVersion: '3.9'
-                  CoverageArg: ''
-            EnvVars:
-              AZURE_SUBSCRIPTION_ID: $(provisioner-subscription)
-              AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-              AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-              AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-              TEST_MODE: 'RunLiveNoRecord'
-              AZURE_SKIP_LIVE_RECORDING: 'True'
-              AZURE_TEST_RUN_LIVE: 'true'
-              REGION: '${{cloud.value.Location}}'
-              # EnvVars for samples to run. TODO: These use an existing FR resource. Fix samples so they don't have to
-              AZURE_FORM_RECOGNIZER_AAD_ENDPOINT: $(python-formrecognizer-test-aad-endpoint)
-              AZURE_FORM_RECOGNIZER_ENDPOINT: $(python-formrecognizer-test-endpoint)
-              AZURE_FORM_RECOGNIZER_KEY: $(python-formrecognizer-test-endpoint-key)
-              CUSTOM_TRAINED_MODEL_ID: $(python-formrecognizer-test-model-trained-without-labels)
-              AZURE_SOURCE_MODEL_ID: $(python-formrecognizer-test-model-trained-with-labels)
-              ID_OF_MODEL_TRAINED_WITH_LABELS: $(python-formrecognizer-test-model-trained-with-labels)
-              ID_OF_MODEL_TRAINED_WITHOUT_LABELS: $(python-formrecognizer-test-model-trained-without-labels)
-              AZURE_FORM_RECOGNIZER_SOURCE_ENDPOINT: $(python-formrecognizer-test-endpoint)
-              AZURE_FORM_RECOGNIZER_SOURCE_KEY: $(python-formrecognizer-test-endpoint-key)
-              AZURE_FORM_RECOGNIZER_TARGET_ENDPOINT: $(python-formrecognizer-test-endpoint)
-              AZURE_FORM_RECOGNIZER_TARGET_KEY: $(python-formrecognizer-test-endpoint-key)
-              AZURE_FORM_RECOGNIZER_TARGET_REGION: $(python-formrecognizer-test-target-region)
-              AZURE_FORM_RECOGNIZER_TARGET_RESOURCE_ID: $(python-formrecognizer-test-target-resource-id)
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      BuildTargetingString: azure-ai-formrecognizer
+      DeployArmTemplate: true
+      ServiceDirectory: formrecognizer
+      TestTimeoutInMinutes: 200
+      TestSamples: true
+      Clouds: 'Prod,Canary'
+      # This is a specific request from the formrecognizer service team
+      # their claim is that the full matrix ends up stress-testing their service.
+      # As such, the canary test runs should run on a reduced matrix.
+      CloudConfig:
+        Prod:
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          Location: 'westus2'
```

Can we use the pipeline variable here? For form recognizer, we switch the prod region quite frequently based on service team recommendation and it's nice to have this configurable when triggering a pipeline.

```suggestion
      Location: $(LOCATION)
```

---

Ah looks like I missed this due to rebase timing. Will update.

---

Thank you!